miso-web/src/spring-test/java/uk/ac/bbsrc/tgac/miso/webapp/springtest/SampleClassRestControllerST.java

```diff
@@ -44,32 +113,66 @@ public void testUpdateNonExistent() throws Exception {
 
   @Test
   @WithMockUser(username = "admin", roles = {"ADMIN"})
-  public void testBulkDelete() throws Exception {
-    List<Long> ids = Arrays.asList(28L);
+  public void testBulkDeleteSuccess() throws Exception {
+    assertNotNull(sampleClassService.get(EXISTING_ID));
 
     getMockMvc().perform(post(CONTROLLER_BASE + "/bulk-delete")
         .contentType(MediaType.APPLICATION_JSON)
-        .content(makeJson(ids)))
+        .content(makeJson(Arrays.asList(EXISTING_ID))))
         .andExpect(status().isNoContent());
   }
 
   @Test
   @WithMockUser(username = "admin", roles = {"ADMIN"})
-  public void testDeleteNonExistent() throws Exception {
-    List<Long> ids = Arrays.asList(999999L);
-
+  public void testBulkDeleteNonExistent() throws Exception {
     getMockMvc().perform(post(CONTROLLER_BASE + "/bulk-delete")
         .contentType(MediaType.APPLICATION_JSON)
-        .content(makeJson(ids)))
+        .content(makeJson(Arrays.asList(999999L))))
         .andExpect(status().isBadRequest());
   }
 
-  private SampleClassDto makeDto(String alias, String suffix) {
-    SampleClassDto dto = new SampleClassDto();
-    dto.setAlias(alias);
-    dto.setSampleCategory("Identity");
-    dto.setSampleSubcategory(null);
-    dto.setSuffix(suffix);
-    return dto;
+  private void assertStatusOrFailWithBody(MvcResult res, int expected) throws Exception {
+    if (res.getResponse().getStatus() != expected) {
+      fail("Expected HTTP " + expected + " but got "
+          + res.getResponse().getStatus()
+          + ". Response body:\n"
+          + res.getResponse().getContentAsString());
+    }
+  }
+
+
+  private SampleClass pickCloneableTemplate() throws Exception {
```

Comment:
It should be clear what data you're testing with. Select specific sample classes.